Package.swift

```diff
@@ -186,3 +184,72 @@ let package = Package(
         )
     ]
 )
+
+hookInternalSwiftSyntaxParser()
+
+/// We need to manually add an -rpath to the project so the tests can run via Xcode
+/// If we are running from console (swift build & friend) we don't need to do it
+func hookInternalSwiftSyntaxParser() {
+    let isFromTerminal = ProcessInfo.processInfo.environment.values.contains("/usr/bin/swift")
```

Comment:
use this approach of having the flag but don't use `/usr/bin/swift`, link to the one that's in the repo as this is proper way to do it, that way the SDK matches the dylib regardless of toolchain user has

Reply:
What should it check then if not '/usr/bin/swift`?

Reply:
we don't really care about user setup for the generic sourcery use, that's why we bundle our own stuff so we don't need to rely, their setup can only affect Swift templates and there are some checks there as far as I remember

Reply:
@krzysztofzablocki I understand that. I have already fixed the path the hook adds to `-rpath` so it points to bundled version of the library. 
But what about `isFromTerminal` flag? You said `/usr/bin/swift` should not be used. So what should we check to determine if `-rpath` should be modified?

Reply:
this use is fine, it just checks if its using swift build but it's not pulling anything from it